packages/blockchain-api/app.mainnet.yaml

```diff
@@ -0,0 +1,12 @@
+runtime: nodejs10
+service: blockchain-api
+env_variables:
+  NODE_ENV: 'production'
+  DEPLOY_ENV: 'mainnet'
+  EXCHANGE_RATES_API: 'https://apilayer.net/api'
+  BLOCKSCOUT_API: 'https://rc1-blockscout.celo-testnet.org/graphql'
+  FIREBASE_PROJECT_ID: 'celo-mobile-mainnet'
+  # TODO Pull addresses from the build artifacts of the network in protocol/build
+  FAUCET_ADDRESS: '0x456f41406B32c45D59E539e4BBA3D7898c3584dA'
+  VERIFICATION_REWARDS_ADDRESS: '0xb4fdaf5f3cd313654aa357299ada901b1d2dd3b5'
```

Comment:
AFAIK the `VERIFICATION_REWARDS_ADDRESS` is not in use (or updated in alfajores vs alfajoresstaging etc) - would it make sense to hardcode this to `0x00000 .... 00verification` or add a comment to make it clear this does not need to be updated

Reply:
Yeah its not used anymore. We can remove it or whatever's easiest.